### DISCUSSION TITLE
Are there any params for delete protection?

### DISCUSSION THREAD

**Discussion by VaZark** [aws/copilot-cli#5582] [Q&A]
Just accidentally deleted a live env of one my minor products (no worries i can have it back in 5 minutes) as the delete option did not ask for the environment when running the command
![image](https://github.com/aws/copilot-cli/assets/11616688/c7cffae6-4f04-4926-bd60-0cb4e9e7b127)

Are there any params to enforce delete protection on a service or environment level (or should i just handle directly on the corresponding CloudFormation stack) ?

**Comment by KollaAdithya** [comment]
Hello @VaZark !

Sorry for the trouble. Looks like a similar enhancement #5211. Can you upvote the issue to help us priortize!

**Reply by VaZark** [reply]
It looks like I've already upvoted #5091 which is referred in turn in #5211 😂

It would be great if that can picked up. This is a relatively minor but annoying and time-consuming error if u have an elaborate setup
